spark/src/test/scala/org/apache/spark/sql/comet/ParquetDatetimeRebaseSuite.scala

```diff
@@ -142,8 +145,13 @@ class ParquetDatetimeRebaseV1Suite extends ParquetDatetimeRebaseSuite {
 class ParquetDatetimeRebaseV2Suite extends ParquetDatetimeRebaseSuite {
   override protected def test(testName: String, testTags: Tag*)(testFun: => Any)(implicit
       pos: Position): Unit = {
-    super.test(testName, testTags: _*)(withSQLConf(SQLConf.USE_V1_SOURCE_LIST.key -> "") {
-      testFun
-    })(pos)
+    // Datasource V2 is not supported by complex readers so force the scan impl back
```

Comment:
Let's be consistent and call them native readers. We will likely use these for more than just complex types at some point.

Reply:
changed the comment